nekoyume/Assets/_Scripts/UI/Module/GrindModule.cs

```diff
@@ -342,11 +345,34 @@ private IEnumerator CoCombineNPCAnimation()
 
             var quote = L10nManager.Localize("UI_GRIND_NPC_QUOTE");
             loadingScreen.AnimateNPC(ItemType.Equipment, quote);
+            loadingScreen.SetCloseAction(() =>
+            {
+                StartCoroutine(CoCrystalGetAnimation(30));
+            });
         }
 
         private void OnNPCDisappear()
         {
             canvasGroup.interactable = true;
         }
+
+        private IEnumerator CoCrystalGetAnimation(int count)
+        {
+            while (count-- > 0)
+            {
+                var anim = Instantiate(crystalMoveAnimation,
+                    MainCanvas.instance.GetLayerRootTransform(WidgetType.Animation));
+
+                anim.Show(
+                    crystalRewardText.transform.position,
+                    Widget.Find<HeaderMenuStatic>().Crystal.IconPosition +
+                    CrystalMovePositionOffset,
+                    Vector2.one,
+                    true,
+                    true,
+                    setMidByRandom: true);
+                yield return null;
+            }
+        }
```

Comment:
Is it okay to instantiate new animation that much?

Reply:
I think so too. It makes many garbage, and will cause frame degradation.
I'll think about ways to improve it.